src/main/scala/TraceIO.scala

```diff
@@ -93,15 +97,18 @@ trait CanHaveTraceIO { this: HasTiles =>
 }
 
 trait CanHaveTraceIOModuleImp extends LazyModuleImp {
```

Comment:
This is probably out of scope for this PR, but should we eventually move to the `InModuleBody` paradigm instead of `ModuleImp`s to mirror rocket-chip?

Reply:
yeah i think so; we can throw out a bunch of extra classes.

Reply:
OK, filed #77